docs/package.json

```diff
@@ -3,13 +3,21 @@
 	"version": "0.0.0",
 	"private": true,
 	"scripts": {
+		"build": "robo build",
+		"deploy": "robo deploy",
+		"dev": "docusaurus serve --dir ./public/ && robox dev",
```

Comment:
This starts two servers and does not build Docusaurus, like one would expect in a `dev` command.

Reply:
It was made for me when I was trying Robo + Docusaurus, but since you have found a new to merge both processes, I will need to discuss how exactly with you so I can push it too